clipmenu

```diff
@@ -57,11 +57,25 @@ else
 fi
 
 [[ $chosen_line ]] || exit 1
-file=$cache_dir/$(cksum <<< "$chosen_line")
-[[ -f "$file" ]] || exit 2
+
+files=()
+while IFS= read -r line; do
+    file=$cache_dir/$(cksum <<< "$line")
+    files+=("$file")
+done <<< "$chosen_line"
+
+paste=()
+for file in "${files[@]}"; do
+    contents=$(cat "$file")
```

Comment:
Command substitution mangles cases with trailing newlines -- take a look at how this was covered in clipmenud for xsel.

Reply:
Will check, thanks.